components/org.wso2.carbon.identity.api.user.session/org.wso2.carbon.identity.api.user.session.v1/src/gen/java/org/wso2/carbon/identity/rest/api/user/session/v1/SessionsApi.java

```diff
@@ -25,10 +25,7 @@
 import org.wso2.carbon.identity.rest.api.user.session.v1.dto.SearchResponseDTO;
 
 import javax.validation.Valid;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
+import javax.ws.rs.*;
```

Comment:
Let's not use * imports.

Reply:
This is a auto generated file. So this should be fine.

Reply:
corrected